### PR TITLE
Add currently active job to worker processes view

### LINKED ISSUE #281: Add current active job to worker processes list
If a worker is currently processing a job it would be awesome if the processes list would show the currently processed job.

With this it would be easier to debug hanging jobs/workers in a setup where multiple workers are present.

### PR DESCRIPTION
**Pull request by LordSimal**
Closes #281 

This adds the currently active job to the process view.

### CODE REVIEW

File: src/Model/Table/QueueProcessesTable.php [hunk 1]
```diff
@@ -59,6 +59,19 @@ public function initialize(array $config): void {
 		$this->setPrimaryKey('id');
 
 		$this->addBehavior('Timestamp');
+
+		$this->hasOne('CurrentQueuedJob', [
```
Comment (dereuromark) [inline src/Model/Table/QueueProcessesTable.php:63]: Assoc are always Plural

Comment (LordSimal) [reply src/Model/Table/QueueProcessesTable.php]: Alright, I have renamed that Assoc now

File: src/Model/Table/QueueProcessesTable.php [hunk 2]
```diff
@@ -59,6 +59,19 @@ public function initialize(array $config): void {
...
+
+		$this->hasOne('CurrentQueuedJobs', [
+				'className' => 'Queue.QueuedJobs',
+				'foreignKey' => 'workerkey',
+				'bindingKey' => 'workerkey',
+				'propertyName' => 'active_job',
+				'conditions' => [
+					'CurrentQueuedJobs.completed IS NULL',
+					'CurrentQueuedJobs.failure_message IS NULL',
+					'CurrentQueuedJobs.failed = 0',
```
Comment (dereuromark) [inline src/Model/Table/QueueProcessesTable.php:71]: Why are those relevant?
it sometimes can be failed 1 and still being re-running.
So I wonder if we could just sync over workerkey config alone?

Comment (LordSimal) [reply src/Model/Table/QueueProcessesTable.php:71]: right, I didn't think of failed jobs which can be re-run.

**Pull request merged**